test/mobile/custom_build/prepare_model.py

```diff
@@ -0,0 +1,17 @@
+import torch
+import torchvision
+import yaml
+
+# Download and trace the model.
+model = torchvision.models.mobilenet_v2(pretrained=True)
+model.eval()
+example = torch.rand(1, 3, 224, 224)
+traced_script_module = torch.jit.trace(model, example)
```

Comment:
Unrelated question: how come we're tracing here (and not using TorchScript, e.g.?)

Reply:
It's copied from a very old example, so the latest approach should be `script_model = torch.jit.script(mode)`?

Reply:
> It's copied from a very old example, so the latest approach should be `script_model = torch.jit.script(mode)`?

This doesn't work - so is there TorchScript MobileNetV2 in torchvision? or are you suggesting I should use some other TorchScript model?